phpunit/class-rest-sidebars-controller-test.php

```diff
@@ -6,6 +6,15 @@
  * @subpackage REST_API
  */
 
+require_once dirname( __FILE__ ) . '/../lib/class-wp-rest-sidebars-controller.php';
+add_filter(
```

Comment:
```suggestion
add_action(
```

Reply:
Also, why do we need to include this here at all? If it is because the endpoint is in an experiment, I think we should do the same thing we did as the block directory and enable the experiment in the tests

https://github.com/tellyworth/gutenberg/commit/26363cfacfe55434dd733705e30937abc34bfbe4#diff-85d18a6eb126c0f8c8da280e74555a2c

In `phpunit/bootstrap.php`.
```php
$GLOBALS['wp_tests_options'] = array(
 	'gutenberg-experiments' => array(
 		'gutenberg-block-directory' => '1',
 	),
 );
```